### PR TITLE
Added workflow to update current-version-tag in README.md

### LINKED ISSUE #972: Fix broken hyperlink in the documentation
**Please provide the issue you face regarding the documentation**
The hyperlink in the below sentence in the `Load a File` section should change from

If the file type is not automatically identified (rare), you can specify them specifically, see section [Specifying a Filetype or Delimiter](https://github.com/capitalone/DataProfiler#specifying-a-filetype-or-delimiter).

to 

If the file type is not automatically identified (rare), you can specify them specifically, see section [Specifying a Filetype or Delimiter](https://capitalone.github.io/DataProfiler/docs/0.10.1/html/profiler.html?highlight=filetype#specifying-a-filetype-or-delimiter).

I am trying to fix the documentation as a way to create my first PR on the repo.

### PR DESCRIPTION
**Pull request by thebadcoder96**
Will fix #972 

As discussed earlier, I have reopened this as a new PR so it can be CLA complaint.

### CODE REVIEW

File: .github/workflows/update-current-version.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    - name: Update README.md
+    
+      run: |
+        latest_tag=$(git describe --tags --abbrev=0)
+        sed "s|<workflow_to_replace_with_live_version_tag>|$latest_tag|g" README.md > tmp.md && mv tmp.md README.md
+        git config --local user.email "mishalksa@hotmail.com"
+        git config --local user.name "GitHub Action"
+        git commit -am "Update README with latest version tag"
+        git push
```
Comment (taylorfturner) [inline .github/workflows/update-current-version.yml:27]: hmm -- nifty what you did here but this creates a situation where each commit that fixes this is tagged to your account, which potentially isn't very robust long-term

**Review by taylorfturner — CHANGES_REQUESTED** [review]

File: .github/workflows/update-current-version.yml (hunk 1 above)
Comment (taylorfturner) [reply .github/workflows/update-current-version.yml:27]: not something that is production-ready per se

Comment (thebadcoder96) [reply .github/workflows/update-current-version.yml:27]: I could create a dummy account and sign the cla. Do you have any ideas on how to make it more robust?

**Comment by taylorfturner** [comment]
Closing after no activity for over a month. Can re-open.

**Pull request closed without merging**